apps/meteor/tests/end-to-end/api/09-rooms.js

```diff
@@ -1141,8 +1142,8 @@ describe('[Rooms]', function () {
 					.end(done);
 			});
 		});
-		it('should return an error when the required parameter "selector" is not provided', (done) => {
-			updatePermission('can-audit', ['admin']).then(() => {
+		(IS_EE ? it : it.skip)('should return an error when the required parameter "selector" is not provided', (done) => {
```

Comment:
I'm still confused why we needed to change this. Why this hadn't failed on develop?

Reply:
It's likely because this EE setting (i.e. `can-audit` ) was earlier getting defined even within CE, so it wasn't failing. But with the changes we did here about moving these settings definitions behind the license, it started to break. Hence we had to update the corresponding tests

Reply:
🤔 but can-audit is not an omnichannel setting, why we moved it?